docs/tutorial/en/source/index.rst

```diff
@@ -34,7 +34,7 @@ Welcome to AgentScope's documentation!
    build_tutorial/visual
    build_tutorial/monitor
    build_tutorial/tool
-   tutorial/rag
```

Comment:
Please remove the original tutorial the file in `tutorial/rag.md`  (both zh and en versions)